engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/graph/node.scala

```diff
@@ -185,7 +184,7 @@ object node {
 
     object SubprocessClazzRef {
 
-      def apply[T: TypeTag]: SubprocessClazzRef = SubprocessClazzRef(ClazzRef.fromDetailedType[T].refClazzName)
+      def apply[T: ClassTag]: SubprocessClazzRef = SubprocessClazzRef(implicitly[ClassTag[T]].runtimeClass.getName)
```

Comment:
WDYT about leaving comment on `SubprocessClazzRef` level that we should also replace this class with `TypingResult`. I know that for now we don't have options on UI to declare that input will be `TypedObjectTypingResult` or `TypedUnion` but can't see any blocker to have such option on BE side. Thanks to that we will have one class less and will be cleaner (we won't mix up compilation (TypingResult) with runtime (classes)).

Reply:
In fact we already have this TODO few lines above :D I added clarification